docs/install.md

```diff
@@ -4,6 +4,7 @@
 
 * [kustomize](https://github.com/kubernetes-sigs/kustomize#kustomize)
 * [oc](https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/)
+* [go](https://github.com/golang/go)
```

Comment:
We should add go version here i.e. s/go/go version 1.12/

Reply:
updated the readme go prerequisite to include go version 1.12 as suggested